sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -164,6 +164,15 @@ kernel_pid_t gnrc_sixlowpan_nd_next_hop_l2addr(uint8_t *l2addr, uint8_t *l2addr_
         nc_entry = gnrc_ipv6_nc_get(iface, next_hop);
     }
     if (ipv6_addr_is_link_local(next_hop)) {
+/* in case of a border router there is no sensible way for address resolution
+ * if the interface is not given */
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER
+        /* if no interface is specified it is impossible to resolve the
+         * link-layer address for a link-local address on a 6LBR */
+        if (iface == NULL) {
+            return KERNEL_PID_UNDEF;
```

Comment:
If this works for your current scenario I'm fine with it. But I think for the future we need some way to assure link-local address translation in some manner we discussed offline. But for now I would leave it as is and focus on it for the next release.